### PR TITLE
Dynamic Copyright Year

### LINKED ISSUE #1878: Addition of Copyright Year in Dashboard footer
Currently, there is no Copyright Year in the footer of Dashboard. 
@RishabhJain2018 @deshraj 

capture

### PR DESCRIPTION
**Pull request by gautamjajoo**
Fixes #1878 
Removed hard coded year in footer and made it dynamic. - GCI
@RishabhJain2018 @vkartik97

### CODE REVIEW

**Comment by vkartik97** [comment]
@gautamjajoo Can you please add the screenshot?

**Comment by gautamjajoo** [comment]
@vkartik97 Here's the screenshot
capture

**Comment by gautamjajoo** [comment]
@RishabhJain2018 Please review.

**Comment by gautamjajoo** [comment]
@RishabhJain2018 Please review the PR and the task.

**Comment by vkartik97** [comment]
@gautamjajoo 
You can define the date object `new Date().getFullYear();` inside the controller onto the `$scope` and then use it in the view to make dynamic updates in the year in AngularJS.

**Comment by gautamjajoo** [comment]
capture

**Comment by gautamjajoo** [comment]
@RishabhJain2018 Please review the PR and the task.

File: frontend/src/js/directives/directives.js [hunk 1]
```diff
@@ -133,14 +133,16 @@
...
+        
+        function controller($scope) {
+            $scope.date = new Date().getFullYear();
+            var js = document.createElement("script");
             js.src = (/^http:/.test(document.location) ? "http" : "https") + "://buttons.github.io/buttons.js";
             document.getElementsByTagName("head")[0].appendChild(js);
        }
     }
 })();
+
```
Comment (RishabhJain2018) [inline frontend/src/js/directives/directives.js]: @gautamjajoo Can you please remove this line?

**Comment by gautamjajoo** [comment]
@RishabhJain2018 Done!

**Comment by gautamjajoo** [comment]
@deshraj Updated!

**Pull request merged**